src/ConstantFlow.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2024, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+#ifndef AZPLUGINS_CONSTANT_FLOW_H_
+#define AZPLUGINS_CONSTANT_FLOW_H_
+
```

Comment:
When I saw this in files, I was trying to keep the includes separate from the defines to make it easier to read, so suggest to add this on its own up here and remove below
```suggestion

#ifndef __HIPCC__
#include <pybind11/pybind11.h>
#endif
```